### ISSUE TITLE
Makes the newly registered domains list subscribable

### ISSUE THREAD

**Issue by privacy-advo** [AdguardTeam/AdGuardDNS#799]
### Issue Details

AdGuard DNS offers the option to block newly registered domains. Please make this list subscribable for AdGuard Windows/AdGuard macOS/AdGuard Home.

**Comment by hagezi** [comment]
@privacy-advo https://github.com/hagezi/dns-blocklists#nrd

**Comment by privacy-advo** [comment]
@hagezi 
Thanks for the fast reply and great to see that there is a published list. Is this the list AdGuard is using?

**Comment by hagezi** [comment]
@privacy-advo I think AdGuard uses similar services to @xRuffkez, but certainly not his list.

**Comment by privacy-advo** [comment]
50 MB probably won't run on a RaspberryPi setup and smaller VPS. Already have AdGuard Home crashes when using your TIF on machines with too little RAM.

**Comment by hagezi** [comment]
![grafik](https://github.com/AdguardTeam/AdGuardDNS/assets/104343908/dac4182f-562d-4171-8c1f-40d0912f87e2)

Works for me on an RPi4 with 4GB RAM. However, the 30-day list also leads to problems when updating all lists.

**Comment by privacy-advo** [comment]
I'm running a Raspberry Pi 3 B+ (1GB RAM) and two VPS (512MB/1GB). On the 512 I really have to limit the lists and on 1GB TIF is fine. The crashes only occure while updating. The issue is known and old https://github.com/AdguardTeam/AdGuardHome/issues/5606. But nothing happened since 2023.

**Comment by hagezi** [comment]
Yes, in "normal operation" my AGH docker container needs around 400MB, when updating the lists the RAM usage goes up to 1.5GB.
